source/includes/stackpath/_workloads.md

```diff
@@ -397,11 +397,11 @@ Edit a workload in a given [environment](#administration-environments).
 
 Required | &nbsp;
 ------- | -----------
-`name`<br/>*string* | The name of the workload.
+`name`<br/>*string* | The name of the workload. The workload name must not exceed 18 characters.
 `type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based. Can be either 'VM' or 'CONTAINER'.
```

Comment:
```suggestion
`type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based. Can be either `VM` or `CONTAINER`.
```

Similar changes can be made at other places as well.

Reply:
good point. 1a15407